src/components/DetailsInputs/DetailsInputs.js

```diff
@@ -68,7 +68,7 @@ const DetailsInputs = ({ inputs }) => {
             filters: { name: dbKey },
             config: {
               params: {
-                iter,
+                iter: iter.includes('@') ? iter.split('@')[0] : iter,
                 tag: hash ?? TAG_FILTER_LATEST
```

Comment:
On many jobs (inputs tub) I see TypeError: Cannot read properties of undefined (reading 'includes')
Also, I saw the same issue with the tag parameter (tag + @tree)
I think we need to Improve a string parsing above